### ISSUE TITLE
Incompatiblity with splunk 6.5 and Mobile Access plugin 2.4.0

### ISSUE THREAD

**Issue by hanneskaiser** [LukeMurphey/lookup-editor#7]
Hi,

we've updated our splunk instance to splunk 6.5 and had some issues with the mobile access add-on  (https://splunkbase.splunk.com/app/2887/) when lookup Editor was installed as well.
According to Splunk Support the lookup editor breaks the mobile access add-on in Splunk 6.5 - there is some incompatibility between both Add-Ons.

The issue could be reproduced by splunk support.
I asked them to inform you about this issue and provide any details.
As I am not sure whether Splunk Support will inform you or provide any additional details, I wanted to make you aware of this issue.

The issue can be reproduced on any splunk 6.5.0 installation that has lookup editor version 2.4.0 and mobile access add-on 2.4.0 installed.
Just try to open the configuration page of the mobile access add-on: It will timeout with some 4xx errors.
Lookup Editor seems to work fine.

Thank you.

**Comment by LukeMurphey** [comment]
Support didn't contact me yet. Could you provide more details on the steps to reproduce the issue?

**Comment by LukeMurphey** [comment]
Oh wait, I think I understand the issue now. Let me try reproducing it.

**Comment by LukeMurphey** [comment]
I got a fix for this. Version 2.5.0 has a fix for this. This isn't the default yet so you will need to manually select it on Splunk-base. Alternatively, you can download it [here](http://lukemurphey.net/attachments/download/397/lookup_editor.tar.gz).

Let me know if that works for you.

**Comment by hanneskaiser** [comment]
Hello Luke,

that was quick, thank you.
I updated our Lookup Editor on Splunk 6.5.0 to latest version 2.5.0 and the Mobile Access Plugin works again. Your fix solves the issue.

Unfortunately I could not provide any additional details before as I did not get any details from Splunk Support myself. Splunk Support just recommended to disable Lookup Editor to make the Mobile Access Plugin work - which is not an option for us, as we need both Add-ons every day.

I'm glad this could be sorted out. 
I will inform Splunk Support as well that the issue has been solved.

Thank you.
